server/client_manager.py

```diff
@@ -204,13 +205,16 @@ def send_area_info(self, area_id, mods):
             info = ''
             if area_id == -1:
                 # all areas info
-                info = '== Area List =='
+                cnt = 0
+                info = '\n== Area List =='.format(len(self.server.area_manager.areas))
```

Comment:
There don't seem to be any replacement fields in this string. The format will have no effect.